modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/util/LabelItem.java

```diff
@@ -40,26 +42,50 @@ public void putData(String key, String value) {
 		data.put(key, value);
 	}
 
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link
+	 *             #setDismissible()}
+	 */
+	@Deprecated
 	public void setCloseable(boolean closeable) {
 		put("closeable", closeable);
```

Comment:
We can remove this, since you should always return `dismissible` instead of `closeable`

Reply:
true

Reply:
@jbalsas mmm how do I always return `dismissible` instead `closeable`? This class extends `HashMap` so if you do `labeItem.get("closeable")` you get closeable, right?

Reply:
Ugh... these APIs we built to work with Soy are not pretty :D 

Nevermind, let's push forward with the tags and figure these details later since in many cases are not so relevant.

Reply:
👍